lib/widgets/cache_image.dart

```diff
@@ -289,8 +291,40 @@ class MixinFileImage extends FileImage {
         controller: controller,
       );
```

Comment:
according https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/painting/image_provider.dart#L529

maybe we can remove function `loadBuffer`?